eth-contracts/contracts/CKBChain.sol

```diff
@@ -69,6 +73,33 @@ contract CKBChain is ICKBChain, ICKBSpv {
 
     /// #ICKBSpv
     function proveTxExist(bytes calldata txProofData, uint64 numConfirmations) external view returns (bool){
+        bytes29 proofView = txProofData.ref(uint40(ViewSpv.SpvTypes.CKBTxProof));
+        uint64 blockNumber = proofView.spvBlockNumber();
+        bytes32 blockHash = proofView.blockHash();
+        require(blockNumber + numConfirmations <= lastBlockNumber, "blockNumber is too ahead");
```

Comment:
`latestBlockNumber ` may be better than `lastBlockNumber`

Reply:
"blockNumber is too ahead" is not clear enough.

Reply:
modify to new msg "blockNumber from txProofData is too ahead of the latestBlockNumber"